web/routers/admin.py

```diff
@@ -30,6 +32,26 @@ async def admin_redirect(request: Request, services: Services = Depends(get_serv
     return RedirectResponse(f"/admin/{available_services[0]}")
 
 
+@router.get("/reset")
+async def reset(request: Request, services: Services = Depends(get_services)):
+    """Show a button to reset the state of the application"""
+
+    return templates.TemplateResponse(
+        "admin/reset.html",
+        {
+            "request": request,
+        },
+    )
+
+
+@router.post("/reset")
+async def post_reset(request: Request, services: Services = Depends(get_services)):
+    """Reset the state of the application"""
+
+    # Restart the application. Note: the state of the application is stored in such a complicated way in memory that it is easier to just restart the application
+    os.execl(sys.executable, sys.executable, *sys.argv)
```

Comment:
yeah, it probably. 

This may do it, but not sure: 
https://github.com/MinBZK/poc-machine-law/blob/main/features/environment.py#L19